src/lumigo_tracer/wrappers/http/http_parser.py

```diff
@@ -61,14 +62,18 @@ def parse_request(self, parse_params: HttpRequest) -> dict:
                 "instance_id": parse_params.instance_id,
             }
 
+        message_id = None
+        if parse_params.headers and is_w3c_headers(parse_params.headers):
+            message_id = get_w3c_message_id(parse_params.headers)
```

Comment:
Where did the message-id come from? Don't we usually get message-id in the response?

Reply:
That's the beauty of OTEL - the parent tells the child what his txid and parentSpanId is :-)